Dockerfile

```diff
@@ -1,16 +1,21 @@
-FROM alpine:3.4
+FROM golang:1.11-alpine
 
 RUN apk update && apk upgrade && \
-     apk add ca-certificates && \
+     apk add ca-certificates \
+     xz-dev \
+     musl-dev \
+     gcc && \
      rm -rf /var/cache/apk/*
 
-RUN mkdir /etc/deployments
+RUN mkdir -p /go/src/github.com/mendersoftware/deployments
 
 EXPOSE 8080
 
-COPY ./config.yaml /etc/deployments/
+COPY . /go/src/github.com/mendersoftware/deployments
+RUN cd /go/src/github.com/mendersoftware/deployments && go install
```

Comment:
I would use `env CGO_ENABLED=1 go install`, just to be explicit, and to have it fail if it's missing or broken. Building without CGO will be something for local development only.